project-setup/base-documentation.md

```diff
@@ -0,0 +1,113 @@
+# Title
+
+Standard base documentation
+
+# Context
+
+A project is to be shared with others as an InnerSource project. In order for
+others to be able to understand what the project is about as well as how to
+contribute, the project needs to provide some base level documentation.
```

Comment:
The context reflects the initial state, the one where we are before the solution (the README) is applied. I think there should be mention that not only is there a need for others to be able to understand what the project is about, but that there is no base level documentation (why that situation exists should be addressed in the Forces section).